one-ui/ui/src/pages/Entities.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { Collapse, Menu } from 'antd';
+import styles from './Entities.module.scss';
+import { RolesContext } from '../util/roles';
+import axios from 'axios'
+import EntityTiles from '../components/entities/entity-tiles';
+
+const Entities: React.FC = () => {
+
+    useEffect(() => {
+        getEntityInfo();
+    },[]);
+
+    const [entitiesInfo, setEntitiesInfo] = useState<any[]>([]);
+    
+    //Role based access
+    const roleService = useContext(RolesContext);
+    const canReadOnly = roleService.canReadMappings();
+    const canReadWrite = roleService.canWriteMappings();
+
+    const getEntityInfo = async () => {
+        try {
+        let response = await axios.get(`/api/entities`);
+        if(response.status === 200) {
+            let entData:any = {};
```

Comment:
I recommend renaming "/api/entities" to "/api/entityTypes" and then using "entityTypes" here instead of "entData" and "entitiesInfo".

Reply:
i renamed entityInfo to entityModel and also changed the corresponding temporary variables to maintain consistency. And since we have been using the api '/api/entities' in current QS as well so we may want to use the same naming conventions here as well right?